### ISSUE TITLE
Mac trackpad unresponsive after installing BTT, works only after quitting BTT

### ISSUE THREAD

**Issue by paksj** [fifafu/BetterTouchTool#824]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Description of bug/feature request/question:
With BetterTouchTool installed, the Macbook trackpad freezes. It does not respond to any click. Other applications (like MS Excel) then hang when any cell/menu bar is clicked. As soon as I quit BTT, everything works fine again. I've tried reinstalling BTT, deleting caches & resetting PRAM, but the problem persists. 

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):
MacBook Trackpad

#### Device information:

Type of Mac: Macbook Pro with TouchBar
macOS version: 10.12.4
BetterTouchTool version: 2.094

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by rich88** [comment]
I'm having this problem also. Every few minutes, the trackpad will go dead - for a few seconds to what seems like 30s. Then it'll come back, right as I'm pulling my hair out. Same device info here. Quitting BTT seems to resolve the issue. Happy to help debug this, as the Touch Bar is pretty useless to me without BTT. :)

**Comment by fifafu** [comment]
This is definitely not normal behavior.  Do you have any other keyboard / trackpad tools installed that may conflict with BTT?

**Comment by paksj** [comment]
No. I only had BTT. I've disabled it now as basic Trackpad is more important than extra gestures/Touch Bar. But like rich88 said, Touch Bar is pretty useless without BTT. Please help.

**Comment by fifafu** [comment]
If possible attach an export of your setup here (using the export preset functionality). Maybe I can spot what's triggering such behavior.


Have you maybe added some heavy Apple Scripts for BTTs Touchbar functionality?

**Comment by kerim** [comment]
I was wondering what was causing my trackpad to be unresponsive. I turned off trackpad gestures in BTT and it seems to have fixed the problem. Was not an issue with earlier versions of BTT, seems to have started with an update that came out about two weeks ago...

**Comment by fifafu** [comment]
I'm currently closing some older issues, in case this still doesn't work correctly, please comment again or open a new issue.

I'll have much more time moving forward as I'm now self-employed. I'm sorry I was not able to solve/reply to this earlier.
